### PR TITLE
Issue 56(new) 

### PR DESCRIPTION
**Pull request by Swastik19Nit**
#85 in resposne to this issue , once again PR raised. Hope it works this time.

### CODE REVIEW

File: Frontend/src/pages/message/message.js [hunk 1]
```diff
@@ -56,25 +59,26 @@ function Message() {
     }
     fetchData();
   }, []);
+
   useEffect(() => {
     if (chatId !== null) {
-      //make a websocket connection here
       let wss = new WebSocket(`wss://mindmate-ws.onrender.com?id=${chatId}`);
+      ws.current = wss;
+      console.log(wss);
```
Comment (afeefuddin) [inline Frontend/src/pages/message/message.js:67]: Remove this `console.log(wss)`

**Comment by afeefuddin** [comment]
There are still bunch of issues with this PR. Made a preview deployment for you to check and fix it.

**Comment by Swastik19Nit** [comment]
image
but the mic feature is actually working in the deployed one. What is the issue which is being faced? @afeefuddin

**Comment by Swastik19Nit** [comment]
@afeefuddin It seems the CSS of the mic was not working. So I am sending one more PR . This time it will definitely work. Thanks for your patience.

**Comment by afeefuddin** [comment]
Don't make a new PR, work on this one only

**Comment by Swastik19Nit** [comment]
@afeefuddin I have changed the CSS here.

**Comment by afeefuddin** [comment]
@subharthihazra Have you tried testing this before merging? This is not working

**Comment by afeefuddin** [comment]
@Swastik19Nit this feature is not working! whenever I click on the mic it does not send any message, I have reverted this PR.

**Comment by afeefuddin** [comment]
I believe this feature is good, but you need to fix it and make a new PR.

**Comment by afeefuddin** [comment]
Okay so there are multiple bugs here 
1) It is not working on brave browser
2) for every new word it is sending a new query that should not happen

Ideal situation is


1) Click on the button and hear the entire message and show it in the text box first,
2) Users can make necessary changes if they want and then hit the send button (We should not send it automatically)

**Comment by Swastik19Nit** [comment]
@afeefuddin yes i can do this . When the message is heard , it will be displayed in text box first and then on hitting the send button, it will be sent . But the problem is speech recognition is not supported in brave.

**Pull request merged**